### PR TITLE
INFODEV-3624

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-a0d593fe61/how-to/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-1c3429130a/how-to/). :bow:

File: content/rackspace-email/enable-or-disable-two-factor-authentication-for-administrators.md [hunk 1]
```diff
@@ -18,31 +18,64 @@ This article describes how to enable two-factor authentication for Cloud Office
...
+
+#### Use text message multi-factor authentication
+
+With text message multi-factor authentication, you can use your phone number to receive a one time code that allows you access to the control panel.
+
+Use the following steps to enable text message multi-factor authentication:
+
+1. Log in to your [Cloud Office Control Panel](https://cp.rackspace.com).
+
+2. When you are prompted with a screen titled **Choose Your Multi-Factor Authentication Method** select the option **Get A Text Message**.
```
Comment (catlook) [inline content/rackspace-email/enable-or-disable-two-factor-authentication-for-administrators.md]: ... Method**, select  ...

File: content/rackspace-email/enable-or-disable-two-factor-authentication-for-administrators.md [hunk 2]
```diff
@@ -18,31 +18,64 @@ This article describes how to enable two-factor authentication for Cloud Office
...
+5. The last step displays the message **You have successfully set up Multi-Factor Authentication!**.
+
+
+#### Use a multi-factor authentication mobile application
+
+Use the following steps to enable a mobile multi-factor authentication application:
+
+1. Log in to your [Cloud Office Control Panel](https://cp.rackspace.com).
+
+2. When you are prompted with a screen titled **Choose Your Multi-Factor Authentication Method** select the option **Use a Mobile App**.
```
Comment (catlook) [inline content/rackspace-email/enable-or-disable-two-factor-authentication-for-administrators.md]: ...  Method**, select ...

File: content/rackspace-email/enable-or-disable-two-factor-authentication-for-administrators.md [hunk 3]
```diff
@@ -18,31 +18,64 @@ This article describes how to enable two-factor authentication for Cloud Office
...
 
 1. Log in to your [Cloud Office Control Panel](https://cp.rackspace.com).
 
 2. Click your username and account number, for example **adminusername (acct#)**, in the upper-right corner to expand the menu.
 
-3. From the menu, select **Enable Two-Factor Auth**.
+3. From the menu, select **Admins & Contacts**.
 
-    <img src="{% asset_path rackspace-email/enable-or-disable-administrator-two-factor-authentication/two_factor_dropdown.png %}"/>
+4. Click the username for the administrator that you are going to require uses two factor authentication.
```
Comment (catlook) [inline content/rackspace-email/enable-or-disable-two-factor-authentication-for-administrators.md]: 4. Click the username for the administrator that you are going to require to use two-factor authentication.

File: content/rackspace-email/enable-or-disable-two-factor-authentication-for-administrators.md [hunk 4]
```diff
@@ -60,12 +93,34 @@ You have successfully logged in using two-factor authentication.
...
+1. Log in to your [Cloud Office Control Panel](https://cp.rackspace.com).
+
+2. Click your username and account number, for example **adminusername (acct#)**, in the upper-right corner to expand the menu.
+
 3. From the menu, select **Admins & Contacts**.
-4. Click the blue username of the admin for whom you need to disable two-factor authentication.
-5. Expand the **Security Settings** section.
-6. Uncheck **Use Two-Factor Authentication**.
+
+4. Click the username for the administrator that you are going to require uses two factor authentication.
```
Comment (catlook) [inline content/rackspace-email/enable-or-disable-two-factor-authentication-for-administrators.md]: 4. Click the username for the administrator that you are going to require to use two-factor authentication.

File: content/rackspace-email/enable-or-disable-two-factor-authentication-for-administrators.md [hunk 5]
```diff
@@ -18,31 +18,64 @@ This article describes how to enable two-factor authentication for Cloud Office
 - **Applies to:** Administrator
 - **Difficulty:** Easy
 - **Time Needed:** Approximately 10 minutes
-- **Tools Needed:** Administrators need access to their Cloud Office Control Panel,
-  their mobile device, and a two-factor authentication application that supports *time-based one-time password* technology
+- **Tools Needed:** Administrators need access to their Cloud Office Control Panel
```
Comment (catlook) [inline content/rackspace-email/enable-or-disable-two-factor-authentication-for-administrators.md]: Note: Throughout the article "multi-factor" should be "multifactor".   I didn't mark all of these.  The exception would be a literal screen heading that still has the hyphen.

"Two factor" should be hyphenated as "two-factor". 

https://www.merriam-webster.com/dictionary/multifactor

**Comment by WilliamLoy** [comment]
@cyrichardson this one is now ready for you :)

**Comment by cyrichardson** [comment]
On it, @WilliamLoy.

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-321f24de42/how-to/)

**Comment by cyrichardson** [comment]
@WilliamLoy Looking good! Suggestions here: 
[enable-or-disable-two-factor-authentication-for-administrators_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2921138/enable-or-disable-two-factor-authentication-for-administrators_cyrEdits.docx)

Please let me know if you have any questions.

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-d08392ca9e/how-to/)

**Pull request merged**